test/projects/file-generate/config/templates/csv-template.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: file-generate-1.0.0
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: "false"
+    description: Test operator
+    containerImage: quay.io/app-sre/file-generate:latest
+    createdAt: "2019-04-10T17:34:33Z"
+    support: Carl Brumm
+spec:
```

Comment:
Do we want this to have a `replaces` or not?

Reply:
I would rather not have a replace as this is used to init a project without former version.

Reply:
Sure: we either include it, and delete it if necessary; or exclude it, and add it if necessary. I don't care either way, but the code and the yaml should all agree :)